roles/edpm_ovn_bgp_agent/tasks/configure.yml

```diff
@@ -22,6 +22,7 @@
 - name: Configure OVN BGP agent
   block:
     - name: Render OVN BGP agent config files
+      become: true
```

Comment:
this also imples that the permission on the dest directory are incorrect 
edpm_ovn_bgp_agent_config_basedir should be writeable by the current user